pages.de/linux/apt.md

```diff
@@ -24,6 +24,10 @@
 
 `sudo apt remove {{paket}}`
 
+- Aktualisiere alle installierten Pakete auf die neuerste Version:
```

Comment:
```suggestion
- Aktualisiere alle installierten Pakete auf die neuste Version:
```

I set it to `neuste` to be consistent with the other file you changed. However, "neueste" is an alternate spelling, which imho sounds better. What would you prefer?

Reply:
`neuste` sounds really funny. But I definitely prefer `neueste`. Also, I didn't use `neuste` in the other page, it was already there and I didn't look close enough. I'll change it.

Reply:
Alright.